src/poker/game/GamePlay.java

```diff
@@ -41,117 +47,113 @@ public GamePlay() {
         this.gameResult = new GameResult();
     }
 
-    // wykryc kto wygra na river (bez all inow) udpate staka i rozpoczęcie nowego handa, potem uprzątnać to co się da
-    // commit i push wysłać linka zadanie3 do git huba
-    public void gameProgress() {
-        int i = 0;
+    public void gameStart() {
+        Player bigBlindPlayer = player1;
+        Player smallBlindPlayer = player2;
         while (playerList.size() > 1) {
-            if (firstHand) {
-            } else {
-                try {
-                    Thread.sleep(4000);
-                } catch (InterruptedException e) {
-                    e.printStackTrace();
-                }
+            System.out.println(bigBlindPlayer.getNamePlayer());
+            System.out.println(bigBlindPlayer.getStackPlayer());
+            System.out.println(smallBlindPlayer.getNamePlayer());
+            System.out.println(smallBlindPlayer.getStackPlayer());
```

Comment:
jak sie zbiera kilka tych printow obok siebie to wynies to do osobnej funkcji np 
printInitPlayersStats(...) albo printPlayersNameAndStack(...)